lib/sequel/plugins/forbid_lazy_load.rb

```diff
@@ -39,6 +39,9 @@ module Plugins
     #
     #   Album.first.artist # no error
     #
+    # This behavior of enabling +forbid_lazy_load+ automatically from dataset
+    # methods can be disabled using the plugin's +:allow_by_default+ option:
```

Comment:
@jeremyevans Ah doh, I just noticed that the colon at the end of this line should be a period. I think I'd intended to put an example under it originally, then decided to put it in the examples section below instead. Probably not worth a PR over, but you may want to do a ninja commit to master or something.

Reply:
Thanks for the heads up. I pushed a commit to fix this.